Subject: [PATCH] bcma: set gpio chip->base for CONFIG_ARCH_BCM_5301X like on 47xx

This makes system GPIOs easier to deal with, e.g. for user space scripts.

Signed-off-by: Felix Fietkau <nbd@openwrt.org>
---
--- a/drivers/bcma/driver_gpio.c
+++ b/drivers/bcma/driver_gpio.c
@@ -240,7 +240,7 @@ int bcma_gpio_init(struct bcma_drv_cc *c
 	 * relative (per chip) numbers.
 	 * So let's use predictable base for BCM47XX and "random" for all other.
 	 */
-#if IS_BUILTIN(CONFIG_BCM47XX)
+#if IS_BUILTIN(CONFIG_BCM47XX) || IS_BUILTIN(CONFIG_ARCH_BCM_5301X)
 	chip->base		= bus->num * BCMA_GPIO_MAX_PINS;
 #else
 	chip->base		= -1;
